test_tf2/test/buffer_core_test.cpp

```diff
@@ -1134,33 +1143,33 @@ TEST(BufferCore_lookupTransform, multi_configuration)
     tf2::BufferCore mBC;
     setupTree(mBC, "1_v", eval_time, interpolation_space);
 
-    if (mBC.canTransform(source_frame, target_frame, eval_time))
+    if (mBC.canTransform(source_frame, target_frame, tf2_ros::fromMsg(eval_time)))
     {
-      geometry_msgs::TransformStamped outpose = mBC.lookupTransform(source_frame, target_frame, eval_time);
-      
+      geometry_msgs::msg::TransformStamped outpose = mBC.lookupTransform(source_frame, target_frame, tf2_ros::fromMsg(eval_time));
+
       if ((source_frame == "1" || source_frame =="2") && (target_frame =="1" || target_frame == "2"))
         EXPECT_TRUE(check_1_result(outpose, source_frame, target_frame, eval_time, epsilon));
-      else if ((source_frame == "a" || source_frame == "b" || source_frame == "c" || source_frame == "f" || source_frame == "g") && 
+      else if ((source_frame == "a" || source_frame == "b" || source_frame == "c" || source_frame == "f" || source_frame == "g") &&
                (target_frame == "a" || target_frame == "b" || target_frame == "c" || target_frame == "f" || target_frame == "g"))
         EXPECT_TRUE(check_v_result(outpose, source_frame, target_frame, eval_time, epsilon));
       else
         EXPECT_FALSE("Frames unhandled");
     }
     else
-      EXPECT_TRUE(((source_frame == "a" || source_frame =="b" || source_frame == "c" || source_frame == "f" || source_frame == "g") && 
+      EXPECT_TRUE(((source_frame == "a" || source_frame =="b" || source_frame == "c" || source_frame == "f" || source_frame == "g") &&
                    (target_frame == "1" || target_frame == "2") )
-                  || 
-                  ((target_frame == "a" || target_frame =="b" || target_frame == "c" || target_frame == "f" || target_frame == "g") && 
+                  ||
+                  ((target_frame == "a" || target_frame =="b" || target_frame == "c" || target_frame == "f" || target_frame == "g") &&
                    (source_frame == "1" || source_frame == "2"))
                   );
-      
-  }      
+
+  }
 }
```

Comment:
Nit: brackets around all of these conditionals, it's a bit confusing as it is now.